packages/dds/sequence/src/test/intervalCollection.spec.ts

```diff
@@ -2098,3 +2100,252 @@ describe("SharedString interval collections", () => {
 		});
 	});
 });
+
+describe("the start and end positions of intervals are updated in response to edits", () => {
```

Comment:
If these are all covering a specific scenario, would it make sense to put them in their own test file?

Reply:
I put them here because I would like to eventually replace many of the tests above, and interval tests are split up between too many files already.
My plan is to gradually grow this test suite to cover insertions as well as other multi-client scenarios. The end state will be that all or almost all of the tests above will be replaced.

I'm not entirely opposed to putting this in its own file, but I think we should do something about the organization of the interval tests first. There's quite a lot of them, and it might make sense to put them in their own subdirectory.